react-native-purchases-ui/ios/RCPaywallFooterViewManager.m

```diff
@@ -0,0 +1,85 @@
+//
+//  RCPaywallFooterViewManager.m
+//  RNPaywalls
+//
+//  Created by Cesar de la Vega on 29/12/23.
+//
+
+#import "RCPaywallFooterViewManager.h"
+@import PurchasesHybridCommon;
+
+#import <React/RCTShadowView.h>
+#import <React/RCTUIManager.h>
+#import <React/RCTBridge.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface CustomFooterView : UIView
+
+- (instancetype)initWithFooterView:(UIView *)footerView bridge:(RCTBridge *)bridge;
+
+@end
+
+NS_ASSUME_NONNULL_END
+
+@interface CustomFooterView ()
```

Comment:
maybe `FooterViewWrapper`? not much `custom` in it

Reply:
Oops yes, that wasn't intentional